### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Application Pool_Error Identification: No Error Identification message is provided for the user even if the user is giving two different inputs in Password and Confirm Password text field.

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#174]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0 , Google Chrome Version 68.0.3440.106 
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS3.3.1, MAS3.3.3

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Web Sites" in the left menu section and Press Enter.
5) Tab till "More" commands button of the existing website inside the grid and Press Enter.
7) Tab till "Edit" in the expanded menu and Press Enter.
8)  Verify that there are proper error messages displayed to the user. Error message should be precise and should tell user exact location and cause of the error for all controls present.

**Actual Result:**
No Error Identification message is provided for the user even if the user is giving two different inputs in Password and Confirm Password text field

**Expected Result:**
User should be able to identify the errors properly. Whenever an user enters any invalid input some error message should be shown to the user about the invalid input and also should explain how to correct the error. 

**User Impact:** 
Users who are blind or have impaired vision will understand easily the nature of the input error and how to correct it.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={733120d0-56c5-479c-889f-fee36ea08ee0}

**Attachments:**
[Bug_174_Error Identification is not provided for the pwd and confirm pwd text field.pptx](https://github.com/Microsoft/IIS.WebManager/files/2332165/Bug_174_Error.Identification.is.not.provided.for.the.pwd.and.confirm.pwd.text.field.pptx)

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK control.

**Comment by jhkimnew** [comment]
This is now fixed.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS3.3.1; #C+E-H; #.NETWebsites; #Chrome; #MAS3.3.3; #Edge

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
